backend/read_data.py

```diff
@@ -28,12 +35,25 @@ def unbias(display_data: pd.DataFrame):
     Returns:
     - unbiased_data: pd.DataFrame
     """
-    pass
+    new_dataframe=display_data.apply(apply_helper_for_unbias, axis=1)
```

Comment:
I haven't seen .apply before but I believe the same mutability issue applies here, just need to make a dr.copy() first